mail_tracking/static/src/js/mail_tracking.js

```diff
@@ -18,7 +18,7 @@ var MessageModel = new Model('mail.message', session.context);
 // chat_manager is a simple dictionary, not an OdooClass
 chat_manager._make_message_super = chat_manager.make_message;
 chat_manager.make_message = function(data) {
-    var msg = this._make_message_super(data);
+    var msg = chat_manager._make_message_super(data);
```

Comment:
AFAICS this wouldn't change anything in reality.

In JS, `this` always belongs to the object being called. To understand better:

```js
a = {c: "pikachu"}
a.b = function () {
  console.log(this.c)  // Logs "pikachu" because `this` is `a`
}
```

So I think the PR is not really needed. Did you really encounter the problem you were describing? Or was it just that it could hypothetically happen? If you encountered, could you test it or provide code to check the bug?

Thanks

Reply:
Please see https://github.com/OCA/social/commit/cac0a83cb42d1ffc6e56ae9883b7d8895e10dc70#diff-c9046c334bb9af935c424584bde259a7R97
where I call the function in order to refresh the thread. This fails without the fix with as shown here:
![image](https://user-images.githubusercontent.com/8435180/38794838-b0b98ba6-4156-11e8-83cd-b05183b92272.png)

Reply:
I agree with @Yajo

Reply:
@chienandalu @Yajo You can see the example I posted. Maybe this is due to Odoo inheritance, when we use `require('mail.chat_manager')`. It seems that other instances don't share the attributes defined here in mail_tracking.